mail_activity_team/models/calendar_event.py

```diff
@@ -0,0 +1,75 @@
+# Copyright 2019 Creu Blanca
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from odoo import api, fields, models, _
+
+
+class CalendarEvent(models.Model):
+    _inherit = 'calendar.event'
+
+    def _get_default_team_id(self, user_id=None):
+        if not user_id:
+            user_id = self.env.uid
+        domain = [('member_ids', 'in', [user_id])]
+        res_model = self.env.context.get('default_res_model')
+        if res_model:
+            model = self.env['ir.model'].search(
+                [('model', '=', res_model)], limit=1)
+            domain.extend(['|', ('res_model_ids', '=', False),
+                           ('res_model_ids', 'in', model.ids)])
+        return self.env['mail.activity.team'].search(domain, limit=1)
+
+    privacy = fields.Selection(selection_add=[
+        ('team', 'Only team'),
+    ])
+    team_id = fields.Many2one(
+        comodel_name='mail.activity.team',
+        default=lambda s: s._get_default_team_id(),
+    )
+
+    def _get_read_fields(self, fields):
+        expected_fields = ['privacy', 'team_id']
+        extra_fields = []
+        fixed_fields = [
+            'id', 'allday', 'start', 'stop', 'display_start', 'display_stop',
+            'duration', 'user_id', 'state', 'interval', 'count',
+            'recurrent_id_date', 'rrule'
+        ]
+        recurrent_fields = self._get_recurrent_fields()
+        public_fields = list(set(
+            recurrent_fields + fixed_fields))
+        if not fields:
+            fields = list(self._fields)
+        for field in expected_fields:
+            if field not in fields:
+                fields.append(field)
+                extra_fields.append(field)
+        return fields, extra_fields, public_fields
+
+    @api.multi
+    def read(self, fields=None, load='_classic_read'):
+        """ This function manages which fields a user can read based on the
```

Comment:
pls use a comment as I mentioned instead. This why you are overriding the original docstring of `read` method.